dataplane/linux/int_dataplane.go

```diff
@@ -1245,19 +1248,34 @@ func (d *InternalDataplane) setUpIptablesBPF() {
 
 		// Do the full RPF check and dis-allow accept_local for anything else.
 		rpfRules = append(rpfRules, rules.RPFilter(t.IPVersion, tc.MarkSeen, tc.MarkSeenMask,
-			d.config.RulesConfig.OpenStackSpecialCasesEnabled, false)...)
+			rulesConfig.OpenStackSpecialCasesEnabled, false)...)
 
 		rpfChain := []*iptables.Chain{{
 			Name:  rules.ChainNamePrefix + "RPF",
 			Rules: rpfRules,
 		}}
 		t.UpdateChains(rpfChain)
 
+		var rawRules []iptables.Rule
+		// Set a mark on the packet if it has come from the WireGuard tunnel to ensure the RPF check allows it
```

Comment:
Comment isn't quite right; you're marking the encapped packets themselves, not the packets coming out of the tunnel

Reply:
I updated the comment, does it seem correct now?